stan/math/prim/fun.hpp

```diff
@@ -126,6 +126,7 @@
 #include <stan/math/prim/fun/hypergeometric_2F2.hpp>
 #include <stan/math/prim/fun/hypergeometric_3F2.hpp>
 #include <stan/math/prim/fun/hypergeometric_pFq.hpp>
+#include <stan/math/prim/fun/hypergeometric_pFq_helper.hpp>
```

Comment:
Do we actually want to include this in prim/fun.hpp? Not fully sure what the implications of leaving it out would be